packages/flutter/lib/src/material/material.dart

```diff
@@ -312,8 +310,8 @@ class Material extends StatefulWidget {
   /// ```dart
   /// MaterialInkController inkController = Material.of(context);
   /// ```
-  static MaterialInkController of(BuildContext context) {
-    final _RenderInkFeatures result = context.findAncestorRenderObjectOfType<_RenderInkFeatures>();
+  static MaterialInkController? of(BuildContext context) {
+    final _RenderInkFeatures? result = context.findAncestorRenderObjectOfType<_RenderInkFeatures>();
     return result;
```

Comment:
nit: this could just be:

```suggestion
return context.findAncestorRenderObjectOfType<_RenderInkFeatures>();
```

Reply:
Done